lib/vagrant-service-manager/command.rb

```diff
@@ -41,16 +41,16 @@ def execute
                 # display information about all the providers inside ADB/CDK
                 self.print_all_provider_info
             else
-                self.print_help
+                self.print_help(1)
```

Comment:
Ok, I understand the exit code of 1 here.

Also, please stop making fun of my favorite ls option :P